### ISSUE TITLE
Paget's disease of bone

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7304]
**Preferred term label:**
Paget's disease of bone

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://pubmed.ncbi.nlm.nih.gov/9661069/

Justified as an HPO term because it is a feature of several Mendelian diseases.

PMID: 23455423
5/5
OMIM:615422 

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
